### ISSUE TITLE
Progress bar displaying COM_CROWDFUNDING_SUCCESSFUL instead of successful

### ISSUE THREAD

**Issue by cityhick** [ITPrism/CrowdFunding#134]
![screenshot](https://cloud.githubusercontent.com/assets/10523781/6042051/d466a498-ac4f-11e4-800a-b0c63033767e.jpg)

http://famousbucks.com/most-funded-campaigns

**Comment by ITPrism** [comment]
Hmm...It seems, the component language is not loaded when the module on other extensions.

You can fix it by loading the component language.

Add the following code at the end of this file "libraries/crowdfunding/init.php".

`$lang->load('com_crowdfunding', CROWDFUNDING_PATH_COMPONENT_SITE);`

**Comment by ssnobben** [comment]
Yes I see this errors before and it would be good if these language strings were added/corrected in future minor updates..rgds nice to see you are doing some progress with crowdfunding!
